### DISCUSSION TITLE
Best method for including dependancies like jQuery?

### DISCUSSION THREAD

**Discussion by nitaic** [hayes0724/shopify-packer#17] [Q&A] [answered]
What is the recommended method to including dependances like jQuery or Boostrap?

Do I add them to the package.json file then reference them in the node_modules directory?

**Comment by GuillermoCasanova** [comment]
I think this is the way, installing them via NPM so they get added to package.json and then import them in the theme.js

**Comment by hayes0724** [comment, marked as answer]
There are several methods I usually install via npm and add to the top of my theme.js file:

```javascript
import jquery from 'jquery';
window.$ = window.jQuery = $
```
